src/models/OtrStore.js

```diff
@@ -24,6 +24,9 @@ function OtrStore(name, client) {
 OtrStore.prototype.initChan = function(args) {
 	var network_id = args.network;
 	var chan = args.chan;
+	var otrStore = this;
+	var client = this.client;
+
 	if (chan.type === Chan.Type.QUERY) {
 		var network = _.find(this.client.networks, {id: network_id});
```

Comment:
Damn you're tough :)

(btw I saaaaaid "This isn't new from this PR" grrrr)

Nevermind then if our entire PR gets denied because of that line, it's on you man, ON YOU!